src/core/parser.ml

```diff
@@ -666,11 +667,11 @@ let parser cmd =
                   Option.get Terms.Defin p,s,al,a,ts_pe)
   | _rule_ r:rule rs:{_:_with_ rule}*
       -> P_rules(r::rs)
-  | e:exposition? _definition_ s:ident al:arg* ao:{":" term}? "≔" t:term
+  | e:exposition? _definition_ st:statement t:{"≔" term}?
         ts_pe:keyword_proof?
-      -> P_definition(Option.get Terms.Public e,false,s,al,ao,t,ts_pe)
-  | e:exposition? st:statement (ts,pe):proof
-      -> P_theorem(Option.get Terms.Public e,st,ts,pe)
+      -> P_definition(Option.get Terms.Public e,false,st,t,ts_pe)
+  | e:exposition? _theorem_ st:statement ts_pe:keyword_proof
```

Comment:
comments from @barras and @fblanqui 

even simpler... theorem is exactly an opaque definition (as it is done in https://github.com/Deducteam/Dedukti) ... so as for a definition, we should be able to give a "proof" lambda-term when the proof is trivial directly like this
```
theorem trivial : typeastheo := triviallambdaproofterm
```

Reply:
fixed in 892ebcb